docs/EN_US/ContainerizedHPCC/ContainerizedMods/TerraformDeployChap.xml

```diff
@@ -0,0 +1,568 @@
+        </informaltable>
+
+        <para>After you clone the terraform-azurerm-hpcc repository you will
+        have access to the modules in that repository.</para>
+
+        <orderedlist>
+          <listitem>
+            <para>Change directory to the virtual network directory
+            first.</para>
+
+            <programlisting>cd terraform-azurerm-hpcc/modules/virtual_network</programlisting>
+          </listitem>
+
+          <listitem>
+            <para>Copy the admin.tfvars files from ./examples to
+            ./virtual_network.</para>
+
+            <para><programlisting>cp examples/admin.tfvars ./admin.tfvars </programlisting></para>
+          </listitem>
+        </orderedlist>
+      </sect3>
+
+      <sect3>
+        <title id="t4MModify_TheModule">Modify the Modules</title>
+
+        <para>To modify the module you can enter the following command
+        (<emphasis role="bold">Note</emphasis> using the code editor in the
+        example, if you prefer you can use nano, vi, or any text
+        editor):</para>
+
+        <para><programlisting>code terraform-azurerm-hpcc/modules/virtual_network/examples/admin.tfvars </programlisting></para>
+
+        <para>With the admin.tfvars file open, you can go through each object
+        block or argument and set it to your preferred values. More
+        information about the module files available in the
+        <emphasis>README.md</emphasis> in HPCC Systems terraform-azurerm-hpcc
+        repository.</para>
+
+        <programlisting>admin = { 
+  name  = "<emphasis role="bold">YourName</emphasis>" 
+  email = "<emphasis role="bold">YourEmail@example.com</emphasis>" 
+        } 
+
+ metadata = { 
+  project             = "hpccdemo" 
+  product_name        = "vnet" 
+  business_unit       = "commercial" 
+  environment         = "sandbox" 
+  market              = "us" 
+  product_group       = "contoso" 
+  resource_group_type = "app" 
+  sre_team            = "hpccplatform" 
+  subscription_type   = "dev" 
+          } 
+ 
+  tags = { "justification" = "testing" } 
+        
+  resource_group = { 
+    unique_name = true 
+    location    = "eastus2" 
+         }     </programlisting>
+
+        <para>From the <emphasis role="bold">virtual network</emphasis>
+        directory:</para>
+
+        <orderedlist>
+          <listitem>
+            <para>Modify this file and replace the values for the name and
+            email fields with your user name and your email.</para>
+          </listitem>
+
+          <listitem>
+            <para>Save the File as admin.tfvars the virtual network
+            directory.</para>
+          </listitem>
+
+          <listitem>
+            <para>Change directory to the<emphasis role="bold"> storage
+            account</emphasis> directory.</para>
+          </listitem>
+
+          <listitem>
+            <para>Copy the admin.tfvars files from ./examples to
+            ./storage_account.  One way you can do this is by issuing the
```

Comment:
This is not told in the same manner as the previous cp command. These should be consistent. (see around line 197)